DemoServer/AdditionalLanguages/python/queries/pagingQueryResults/metadata.json

```diff
@@ -0,0 +1,140 @@
+{
+    "Slug": "paging-query-results",
+    "SourceFileName": "pagingQueryResults.py",
+    "Title": "Paging Query Results",
+    "DescriptionLines": [
+        "By default, a query will return _all_ available matching results.",
+        "**Paging** can be used to limit the result set and avoid handling huge amounts of data at once.",
+        "",
+        "Page the results by specifying the number of results to skip and the number of results to take.",
+        "Either loop page by page on all available results, or request a specific page out of all results."        
+    ],
+
+    "Assets": [
+        {
+            "Title": "Deep paging",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#sorting-query-results"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'Companies' collection using the Session `query()` method.",
+                "",
+                "* While this example relates to a dynamic query where an auto-index is created,<br>",
+                "  _paging_ can also be applied when querying using a static index."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/python/queries/query-overview"
+                }
+            ]
+        },
+        {
+            "Title": "Get query statistics",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* When paging results, the query returns only the requested number of documents.<br>",
+                "  See step-4.",
+                "",
+                "* Using the `statistics()` method allows retrieving the number of total available matching results.",
```

Comment:
i think this may be a good place to also mention, that the statistics method takes a callback function

Reply:
done, added:
=>
The statistics method expects a callback function, which receives a single `QueryStatistics` parameter.